db.json

```diff
@@ -0,0 +1,50 @@
+[
+    {
+      "destination": "Costa Rica",
+      "from": "XX.XX.XXXX",
+      "to": "XX.XX.XXXX",
+      "entryTitle": "impresionante ",
+      "journalEntry": "Lorem ipsum dolor sit ...",
+            "id": "0"
+    },
+    {
+        "destination": "Kuba",
+        "from": "XX.XX.XXXX",
+        "to": "XX.XX.XXXX",
+        "entryTitle": "Lindo",
+        "journalEntry": "Lorem ipsum dolor sit ...",
+              "id": "1"
+    },
```

Comment:
Why is this file not properly formatted? Is your prettier running?

Reply:
i think so...but i have the feeling that my vsc has some issues...